etc/input.conf

```diff
@@ -31,6 +31,7 @@
 #MBTN_LEFT     ignore              # don't do anything
 #MBTN_LEFT_DBL cycle fullscreen    # toggle fullscreen
 #MBTN_RIGHT    cycle pause         # toggle pause/playback mode
+#MBTN_MID      script-binding positioning/align-to-cursor # pan the whole image
```

Comment:
The assumption is what https://github.com/mpv-player/mpv/pull/15320 will merged first, so it won't do anything by default until you zoom in.

Reply:
We could use `MBTN_MID_DBL` to reset. I agree though there should be a way to undo the change with mouse only.

Browsers also enter panning mode with middle mouse button, without modifiers.

Reply:
Bound `MBTN_MID_DBL` to reset align and pan. When `MBNT_RIGHT` will `drag-to-pan` I guess can we also bind `MBTN_RIGHT_DBL` to reset?

Reply:
MBTN_MID can also be bound only for images, if preferred by users to not affect panscan, though I think it can be useful to pan after panscan and zooming videos.